src/main/java/no/finn/unleash/strategy/StrategyUtils.java

```diff
@@ -0,0 +1,55 @@
+package no.finn.unleash.strategy;
+
+public final class StrategyUtils {
+    private static final int ONE_HUNDRED = 100;
+
+    public static boolean isNotEmpty(final CharSequence cs) {
+        return !isEmpty(cs);
+    }
+
+    public static boolean isEmpty(final CharSequence cs) {
+        return cs == null || cs.length() == 0;
+    }
+
+    public static boolean isNumeric(final CharSequence cs) {
+        if (isEmpty(cs)) {
+            return false;
+        }
+        final int sz = cs.length();
+        for (int i = 0; i < sz; i++) {
+            if (Character.isDigit(cs.charAt(i)) == false) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    /**
+     * Takes to string inputs concat them, produce a hashCode and return a normalized value between 0 and 100;
+     *
+     * @param identifier
+     * @param groupId
+     * @return
+     */
+    public static int getNormalizedNumber(String identifier, String groupId) {
+        int hashCode = Math.abs((groupId + ':' + identifier).hashCode());
```

Comment:
I got this warning once upon a time http://stackoverflow.com/questions/23416264/bad-attempt-to-compute-absolute-value

Can be changed to something like:
```
final int hashCode = (identifier + groupId).hashCode();
return 1 + Math.abs(hashCode % ONE_HUNDRED);
```